### ISSUE TITLE
[svg-native] `overflow: scroll` is both forbidden and to behave like `hidden`

### ISSUE THREAD

**Issue by maxnordlund** [w3c/svgwg#687]
I noticed this contradiction in [masking](https://svgwg.org/specs/svg-native/#masking).

> The scroll value on the overflow property behaves identically to hidden.
> ...
> The scroll and the auto values of the overflow property are forbidden.

I assume it's supposed to be one or the other?

**Comment by svgeesus** [comment]
Interesting. The [initial value of `overflow` is `visible`](https://drafts.csswg.org/css-overflow-3/#propdef-overflow). 

If scroll and auto are forbidden values, one would assume that they are not allowed in conforming content, and that is a conforming renderer encounters them they are ignored (== treated as the initial value).

If that is what we want, we can fudge around things by setting a 'UA stylesheet' that sets `overflow: hidden` on the `svg` element. Which would be inline with the [special rues and uA stylesheet for overlow in SVG2](https://www.w3.org/TR/SVG2/render.html#OverflowAndClipProperties). This effectively sets the initial value to `hidden` on SVG content. Which is fine, as long as we disallow the `initial` keyword.

**Comment by maxnordlund** [comment]
Yeah, that makes sense. The [styling](https://svgwg.org/specs/svg-native/#styling) chapter already forbids `intial`, so that's already taken care of.

> All the global CSS keywords are forbidden. These include:
> 1. initial
> ...

Perhaps it's best to rely on SVG2 here, or at least make a note to do so when #661 is worked on.

**Comment by dirkschulze** [comment]
I suppose this is a similar issue as for `gradientUnits` for gradients: default is `objectBoundingBox` but SVG Native only supports `userSpaceOnUse`. Maybe the wording should be tweaked.
